web3walletcore/src/commonMain/kotlin/com/sonsofcrypto/web3walletcore/services/actions/Action.kt

```diff
@@ -1,25 +1,28 @@
 package com.sonsofcrypto.web3walletcore.services.actions
 
-import com.sonsofcrypto.web3walletcore.services.actions.Action.Type.*
+import com.sonsofcrypto.web3walletcore.extensions.Localized
 
-data class Action(
-    val type: Type,
-    val imageName: String, // Security, Social, etc
+sealed class Action(
     val title: String,
     val body: String,
-    val deepLink: String,
-    val canDismiss: Boolean,
-    val order: Int, // 1, 2, 3, 4...(left to right)
+    val imageName: String,
 ) {
-    sealed class Type {
-        data class ConfirmMnemonic(val address: String): Type()
-        object Themes: Type()
-        object ImprovementProposals: Type()
 
-        val identifier: String get() = when (this) {
-            is ConfirmMnemonic -> "modal.mnemonic.confirmation.${address}"
-            is Themes -> "settings.themes"
-            is ImprovementProposals -> "modal.improvementProposals"
-        }
-    }
+    object ConfirmMnemonic: Action(
+        Localized("actions.mnemonic.title"),
+        Localized("actions.mnemonic.body"),
+        "s",
+    )
+
+    object Themes: Action(
+        Localized("actions.themes.title"),
+        Localized("actions.themes.body"),
+        "t",
+    )
+
+    object ImprovementProposals: Action(
+        Localized("actions.improvementProposals.title"),
+        Localized("actions.improvementProposals.body"),
+        "i",
+    )
 }
```

Comment:
More kotlin native way to to model it would be like so.
We need to localize every string